app/env/Environment.dev.ts

```diff
@@ -1,8 +1,32 @@
+function getBitcoinAddressFormat(): string {
+  return "([13]|bc1)[a-zA-HJ-NP-Z0-9]{25,62}";
+}
+
+function getLightningAddressFormat(): string {
+  return "(LNURL|LNDHUB)[A-Z0-9]{25,250}|LNNID[A-Z0-9]{66}";
+}
+
+function getEthereumAddressFormat(): string {
+  return "0x\\w{40}";
+}
+
+function getDefichainAddressFormat(): string {
+  return "[78]\\w{33}|[td]\\w{33}|[td]\\w{41}";
+}
+
+function getAllAddressFormat(): string {
+  const bitcoin = getBitcoinAddressFormat();
+  const lightning = getLightningAddressFormat();
+  const ethereum = getEthereumAddressFormat();
+  const defichain = getDefichainAddressFormat();
+
+  return `${bitcoin}|${lightning}|${ethereum}|${defichain}`;
+}
```

Comment:
Maybe also use variables instead of functions?

Reply:
Taproot/multisig addresses missing?

Reply:
taproot and multisig is contained in Bitcoin Address Format: "([13]|bc1)[a-zA-HJ-NP-Z0-9]{25,62}"